packages/tsx-editor/README.md

```diff
@@ -27,7 +27,7 @@ If the user's browser can't support the editor (mainly IE 11 and some mobile bro
 Any project consuming `@uifabric/tsx-editor` should follow the Webpack and runtime configuration instructions from the [`@uifabric/monaco-editor` readme](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/monaco-editor/README.md). Note that the helpers used are re-exported from this package for convenience:
 
 - `addMonacoWebpackConfig`: import from `@uifabric/tsx-editor/scripts/addMonacoWebpackConfig`
-- `configureEnvironment` and `IMonacoConfig`: import from `@uifabric/tsx-editor`
+- `configureEnvironment` and `IMonacoConfig`: import from `@uifabric/tsx-editor/lib/index-min`
```

Comment:
this seems like not an improvement. Can we avoid suggesting to import path based things?

Reply:
tsx-editor is kind of a special case because the normal `index` file exports all the public APIs, including lower-level ones which directly import Monaco. So other packages shouldn't directly import from that file because:
1. It greatly increases bundle size (Monaco is huge)
2. It will break Jest tests, even if the editor isn't instantiated (Monaco uses ES modules, which Jest *still* hasn't added support for, so even trying to parse it makes Jest blow up)

My solution to this was to make a separate `index-min` file which doesn't reference Monaco aside from types, but I'm open to other suggestions--maybe reversing it somehow, but I'm not sure what the new file should be called (`index-max` just sounds silly).